bigtable/tests/integration_tests_utils.sh

```diff
@@ -56,10 +56,10 @@ function run_all_integration_tests() {
 
   echo
   echo "Running bigtable::InstanceAdmin integration test."
-  if [ -z "${BIGTABLE_INSTANCE_ADMIN_EMULATOR_HOST}" ]; then
+  if [ -z "${BIGTABLE_INSTANCE_ADMIN_EMULATOR_HOST:-}" ]; then
     ./instance_admin_integration_test "${project_id}";
   else
-    env BIGTABLE_EMULATOR_HOST=${BIGTABLE_INSTANCE_ADMIN_EMULATOR_HOST} \
+    env "BIGTABLE_EMULATOR_HOST=${BIGTABLE_INSTANCE_ADMIN_EMULATOR_HOST:-}" \
```

Comment:
Did you mean to put this entire NAME=VALUE section in quotes?  I assume it's only supposed to have the VALUE portion quoted.

Reply:
Both have the same effect, right?  We have been quoting the full expression in other scripts, but happy to change it.

Reply:
I did a test in my local terminal and... was testing the wrong thing :)  This totally makes sense, since this is one whole arg going to `env`.